packages/expo-cli/scripts/preversion.js

```diff
@@ -0,0 +1,49 @@
+let { bold } = require('chalk');
+let boxen = require('boxen');
+let inquirer = require('inquirer');
+
+console.log(
+  boxen(bold("Please complete these checks before publishing the 'expo-cli' package:"), {
+    padding: 1,
+  }) +
+    `
+${bold('1) Offline development')}:
+  Check that you're logged in (\`expo whoami\`) and then make sure
+  \`expo start\` works without internet connection.
```

Comment:
If `expo start --offline` is the correct workflow for offline development, could you change the text here to reflect that, as well?